BlinkStickDotNet/AssemblyInfo.cs

```diff
@@ -4,10 +4,10 @@
 // Information about this assembly is defined by the following attributes. 
 // Change them to the values specific to your project.
 
-[assembly: AssemblyTitle("BlinkStick.Hid")]
-[assembly: AssemblyDescription("BlinkStick Generic Hid Library")]
+[assembly: AssemblyTitle("BlinkStick HID")]
+[assembly: AssemblyDescription("BlinkStick Generic Hid Library with KeesTalksTech modifications")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Agile Innovative Ltd")]
+[assembly: AssemblyCompany("KeesTalksTech")]
```

Comment:
Has been a while. I did it to see the difference between the projects (the DLL from the Nuget package is not an official DLL).